iree/hal/dylib/cts/CMakeLists.txt

```diff
@@ -4,6 +4,15 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+if (IREE_BYTECODE_MODULE_FORCE_SYSTEM_DYLIB_LINKER)
+  set (IREE_HAL_DYLIB_CTS_EXECUTABLE_FORMAT_PREFIX "system")
```

Comment:
nit: local/internal variables (that aren't used from other CMakeLists) can use shorter names (vs these fully qualified names that read like options for users to set), optionally prefixed with `_`
`_EXECUTABLE_FORMAT_PREFIX` and `_EXECUTABLE_FORMAT` would do just fine here